package_managers/install_pkgs.bzl

```diff
@@ -14,7 +14,25 @@
 
 """Rule for installing apt packages from a tar file into a docker image."""
 
+load("//package_managers:package_manager_provider.bzl", "package_manager_provider")
+
 def _impl(ctx):
+  package_manager = ctx.attr.package_manager_generator[package_manager_provider]
+  # The defualt order for depset is deterministic and hence we should always see the tar
+  # at index 0.  See https://docs.bazel.build/versions/master/skylark/lib/depset.html
+  installables_tar = ctx.attr.package_manager_generator.default_runfiles.files.to_list()[0]
+  # Generate the installer.sh script
+  install_script = ctx.new_file("%s.install" % (ctx.label.name))
+  ctx.template_action(
+      template=ctx.file._installer_tpl,
+      substitutions= {
+          "%{apt_get_install_commands}": '\n'.join(package_manager.install_commands),
```

Comment:
Should this be renamed to just "install_commands" since this is now generic?

Reply:
done